src/components/Project/Secrets/paginator.js

```diff
@@ -400,6 +401,22 @@ export default class SecretsPaginator extends React.Component {
     this.openDrawer()
   };
 
+  onPanelTableEmpty() {
+    console.log("onPanelTableEmpty!")
+    return (
+      <Card square={true}>
+        <CardContent>
+          <Typography variant="subheading" style={{ textAlign: "center", fontWeight: 500, fontSize: 23, color: "gray" }}>
+            No Secrets Found!
```

Comment:
This is unrelated to the service view changes, correct?

Reply:
No, it's there in case you put a dumb filter in and you have no results. It's a reminder that you have a filter set.